lib/ansible/modules/cloud/ovirt/ovirt_vms.py

```diff
@@ -165,7 +165,7 @@
         description:
             - Type of the Virtual Machine.
             - Default value is set by oVirt/RHV engine.
-        choices: [ desktop, server ]
```

Comment:
I would add here (right after `- Default value is set by oVirt/RHV engine.` ):
```
- I(high_performance) is supported since Ansible 2.5 and oVirt/RHV 4.2.
```